### PR TITLE
[OPT-54] to OPT-2

### PR DESCRIPTION
**Pull request by dantolin-iriusrisk**

### CODE REVIEW

File: startleft/cli.py [hunk 1]
```diff
@@ -129,9 +118,15 @@ def threatmodel(recreate, irius_server, api_token, filename):
     """
     Uploads an OTM file to IriusRisk
     """
-    otm_to_ir = OtmToIr(irius_server, api_token)
-    logger.info("Uploading OTM file and generating the IriusRisk threat model")
-    otm_to_ir.run(recreate, filename)
+    otm_project = OtmProject.from_otm_file(filename)
+    otm_service = OtmProjectService(IriusriskProjectRepository(api_token, irius_server))
```
Comment (pablosantamartae) [inline startleft/cli.py:122]: From this line till the end of this cli command, the code is exactly the same as in **_run_** cli command. Could be a good idea to create a common method for that.

File: startleft/cli.py [hunk 2]
```diff
@@ -81,21 +78,16 @@ def run(type, map, otm, name, id, recreate, irius_server, api_token, filename):
     """
     Parses IaC source files into Open Threat Model and immediately uploads threat model to IriusRisk
     """
+    logger.info("Parsing IaC source files into OTM")
+    otm_project = OtmProject.from_iac_file(id, name, type, filename, map, otm)
+    otm_service = OtmProjectService(IriusriskProjectRepository(api_token, irius_server))
```
Comment (pablosantamartae) [inline startleft/cli.py:83]: From this line till the end of this cli command, the code is exactly the same as in **_threadmodel_** cli command. Could be a good idea to create a common method for that.

File: startleft/project/iriusrisk_project_repository.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from startleft.apiclient.calls.create_project import CreateProject
+from startleft.apiclient.calls.delete_project import DeleteProject
+from startleft.apiclient.calls.update_project import UpdateProject
+from startleft.mapping.otm_file_loader import OtmFileLoader
+from startleft.validators.otm_validator import OtmValidator
+
+logger = logging.getLogger(__name__)
+
+
+def get_project_id(otm_file: str) -> int:
```
Comment (pablosantamartae) [inline startleft/project/iriusrisk_project_repository.py:14]: Is this method necessary? I didn't find any references of it in the code.

**Comment by dantolin-iriusrisk** [comment]
The origin branch of this PR was named OPT-51 because of a mistake. Actually, it is related to the task OPT-54, so the PR has been renamed to match this task ID.

**Pull request merged**